frontend/src/views/DashboardView.vue

```diff
@@ -1,7 +1,7 @@
 <script setup lang="ts">
-import Dashboard from '@/components/dashboard/Dashboard.vue';
+import ManagePermissions from '@/components/managePermissions/ManagePermissions.vue';
 </script>
 
 <template>
-    <Dashboard />
+    <ManagePermissions />
```

Comment:
Wondering why you still keep "DashboardView.vue" while it seem entirely been replaced by ManagePermissions.vue?
I am not aware we are anticipating some other components or extra styling or logic here, or will we?

Reply:
I think I missed this one. Thanks!